src/plugins/condition/components/inspector/ConditionalStylesView.vue

```diff
@@ -106,6 +106,7 @@ import ConditionError from "@/plugins/condition/components/ConditionError.vue";
 import Vue from 'vue';
 import PreviewAction from "@/ui/preview/PreviewAction.js";
 import {getApplicableStylesForItem} from "@/plugins/condition/utils/styleUtils";
+import _ from 'lodash';
```

Comment:
How was this working without the import statement?

Reply:
I don't know, but in this case it is no longer needed as I've replaced the lodash isEmpty with the native implementation in arrayUtils.

Reply:
I think lodash has leaked onto window somehow.